app/cdap/components/ParsingDrawer/index.tsx

```diff
@@ -0,0 +1,91 @@
+import React, { useEffect, useState } from 'react';
+import Box from '@material-ui/core/Box';
+import ErrorOutlineIcon from '@material-ui/icons/ErrorOutline';
+import { Button } from '@material-ui/core';
+import { useStyles } from './styles';
+import { APPLY_BUTTON, IMPORT_SCHEMA, PARSING, PARSING_INFO_TEXT } from './constants';
+import ParsingPopupBody from './Components/ParsingPopupBody';
+import DrawerWidget from 'components/DrawerWidget';
+import ParsingHeaderActionTemplate from './Components/ParsingHeaderActionTemplate';
+
+const ParsingDrawer = (props) => {
+  const [drawerStatus, setDrawerStatus] = useState(true);
+  const [formatValue, setFormatValue] = useState();
+  const [encodingValue, setEncodingValue] = useState();
+  const [quotedValuesChecked, setQuotedValuesChecked] = useState(false);
+  const [headerValueChecked, setHeaderValueChecked] = useState(false);
+  const classes = useStyles();
+
+  useEffect(() => {
+    setDrawerStatus(true);
+  }, []);
+
+  const closeClickHandler = () => {
+    setDrawerStatus(false);
+  };
+
+  const handleFormatChange = (event: React.ChangeEvent<HTMLSelectElement>) => {
+    const value = event.target.value as any;
+    setFormatValue(value);
+  };
+
+  const handleEncodingChange = (event: React.ChangeEvent<HTMLSelectElement>) => {
```

Comment:
remove React. and import at the top of the file

Reply:
done